Makefile

```
@@ -283,3 +286,33 @@ COMMIT ?= "HEAD"
 add-tags: | $(MULTIMOD)
 	@[ "${MODSET}" ] || ( echo ">> env var MODSET is not set"; exit 1 )
 	$(MULTIMOD) verify && $(MULTIMOD) tag -m ${MODSET} -c ${COMMIT}
+
+# The source directory for opentelemetry-configuration schema.
+OPENTELEMETRY_CONFIGURATION_JSONSCHEMA_SRC_DIR=tmp/opentelememetry-configuration
+
+# The SHA matching the current version of the opentelemetry-configuration schema to use
+OPENTELEMETRY_CONFIGURATION_JSONSCHEMA_VERSION=v0.1.0
+
+# Cleanup temporary directory
+genjsonschema-cleanup:
+	rm -Rf ${OPENTELEMETRY_CONFIGURATION_JSONSCHEMA_SRC_DIR}
+
+GENERATED_CONFIG=./config/pkg/${OPENTELEMETRY_CONFIGURATION_JSONSCHEMA_VERSION}/generated_config.go
```

> Note that the generated model code lives in a package that includes the version of the schema in the package name.
I'm not sure if that's the right thing to do, but it is similar to what is done for semconv.

With semconv we indented to support all kind of versions. For the configuration, I suppose it is OK to support the latest configuration schema version. Therefore, I do not think we should include a version in the package name. Similar PR: https://github.com/open-telemetry/opentelemetry-go/pull/4538

```suggestion
GENERATED_CONFIG=./config/model.go
```